topics/modules-api-ref.md

```diff
@@ -7588,7 +7684,7 @@ Returns `VALKEYMODULE_OK`.
 
 ### `ValkeyModule_GetFunctionExecutionState`
 
-    ValkeyModuleScriptingEngineExecutionState ValkeyModule_GetFunctionExecutionState(;
+    ValkeyModuleScriptingEngineExecutionState ValkeyModule_GetFunctionExecutionState( ValkeyModuleScriptingEngineServerRuntimeCtx *server_ctx);
```

Comment:
This is an annoying artifact, but it's generated by the ruby code and I don't really want to fix it tonight. It's currently completely wrong, so would rather followup tomorrow the fix.